deps/juggler-v3/test.js

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: loopback-connector-cloudant
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+const juggler = require('loopback-datasource-juggler');
+const name = require('./package.json').name;
+
+require('../../test/init.js');
+
+describe(name, function() {
+  before(function() {
+    global.IMPORTED_TEST = true;
+    return global.resetDataSourceClass(juggler.DataSource);
+  });
+
+  after(function() {
+    global.IMPORTED_TEST = false;
+    return global.resetDataSourceClass();
+  });
+  require('loopback-datasource-juggler/test/include.test.js');
+  require('loopback-datasource-juggler/test/common.batch.js');
+});
```

Comment:
Please add a TODO comment to remind us that we should eventually include the following tests too:
- `loopback-datasource-juggler/test/default-scope.test.js`
- `loopback-datasource-juggler/test/persistence-hooks.suite.js`

See https://github.com/strongloop/loopback-connector-mongodb/blob/master/deps/juggler-v4/test.js